templates/pages/admin/form/form_destination_commonitil_config.html.twig

```diff
@@ -78,10 +78,4 @@
             )|raw }}
         </section>
     {% endfor %}
-
 </div>
-
-<script src="{{ js_path("js/form_destination_auto_config_controller.js") }}"></script>
-<script>
-    new GlpiFormDestinationAutoConfigController();
-</script>
```

Comment:
Moved into `templates/pages/admin/form/form_destination.html.twig` to make sure the script is loaded once as the previous template was loaded for each existing destination.